projects/plugins/backup/src/js/components/rna-styles.scss

```diff
@@ -0,0 +1,183 @@
+/********* RNA styles *********/
+// We should extract this into a package
```

Comment:
How much of this could be pulled directly from `@wordpress/base-styles` and from `@automattic/color-studio`?

For Jetpack blocks we pull both in our build, for example:
- https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/extensions/shared/styles/gutenberg-base-styles.scss#L1-L6
- https://github.com/Automattic/jetpack/blob/a9b89148a442d7fd3686ade9e9c015ad5a65807a/projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss#L2
- https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/extensions/shared/block-icons.js#L4

Reply:
Tracked in #20334.